assets/js/common/AbilitiesMultiSelect/AbilitiesMultiSelect.jsx

```diff
@@ -52,7 +52,10 @@ const groupedAbilities = [
   {
     ability: 'operation:all',
     tooltip: 'Permits running operations in all resources',
-    groupAbilities: ['saptune_solution_apply:host'],
+    groupAbilities: [
+      'saptune_solution_apply:host',
```

Comment:
not your fault at all, but I will need to fix many conflict resolutions in my `cluster_maintenance_change` code that is based in other PRs that I cannot merge due the merge freeze.
just to release my frustration back again :sweat_smile: